app/src/main/java/org/simple/clinic/signature/SignatureEffect.kt

```diff
@@ -0,0 +1,3 @@
+package org.simple.clinic.signature
```

Comment:
Just an FYI. When we are working on new features, we should not implement no-op Mobius classes. We should be implementing code as needed.

Reply:
We'll make sure the next time. Thanks